src/pages/challenge-specifics/component.js

```diff
@@ -45,7 +74,7 @@ export const ChallengeSpecifics = ({location, voteChallenge, user, clearChalleng
         defaultActiveKey={location.state.defaultTab}
       >
         <TabPane tab="Activities" key={CNST.ROUTES.CHALLENGE_ACTIVITIES_TAB}>
-          <ChallengeActivities />
+          <ChallengeActivities challengeId={challengeId} />
```

Comment:
Looks like you did not pass all props.
Because we have another view for the owner
![image](https://user-images.githubusercontent.com/82516098/130191861-c8d0c4fb-f53b-4adb-9821-b0be871deec3.png)